src/components/Content/Pursuance/PursuancePage.js

```diff
@@ -9,31 +9,76 @@ import DiscussView from './views/DiscussView';
 import ParticipantsView from './views/ParticipantsView';
 import RightPanel from '../RightPanel/RightPanel';
 import './PursuancePage.css';
+import ReactModal from 'react-modal';
 
 class PursuancePage extends Component {
-
+ constructor(){
+  super();
+  this.state={
+    modalIsOpen : false
+   }
+ }
   componentWillMount() {
     let { setCurrentPursuance, match, currentPursuanceId } = this.props;
     currentPursuanceId = Number(match.params.pursuanceId) || currentPursuanceId;
     setCurrentPursuance(currentPursuanceId);
+    ReactModal.setAppElement('body');
```

Comment:
@greg-han What does this do?

Reply:
This mounts the react modal to the body.
This should be default within the modal but i guess it's a known bug in the library.
If I don't add that snippet when the component mounts, I get console errors related to accessing the modal.